Gemfile

```diff
@@ -103,6 +103,13 @@ gem "view_component"
 # Catching unsafe migrations in development
 gem "strong_migrations"
 
+# TODO: Added on 12 Nov 2024 - CB
```

Comment:
could add ticket number AP-5513 here for completenes

Reply:
It didn't exist when I wrote the comment! :D 
Will get it added 👍